xbmc/addons/IAddon.h

```diff
@@ -27,7 +27,11 @@ using AddonInstanceId = uint32_t;
 enum class AddonOptPostInstValue
 {
   //! Default value if unused on add-on type.
-  UNUSED = -1
+  UNUSED = -1,
```

Comment:
Is -1 required here? Can it be omitted and left as 0 to avoid mixing enums and ints, or is -1 important?